aiohttp_sse/__init__.py

```diff
@@ -25,7 +29,7 @@ def hello(request):
 
     DEFAULT_PING_INTERVAL = 15
 
-    def __init__(self, *, status=200, reason=None, headers=None):
+    def __init__(self, *, status=200, reason=None, headers=None, request=None):
```

Comment:
Good point, I just trying to find a way to have `EventSourceResponse` without `request` in parameters, since class inherited from `aiohttp. StreamResponse` we should try hard to keep interface/abstraction, currently no `Response` derived class takes `request` as parameter.... May be we should follow websocket interface?
aiohttp case:
```python
async def websocket_handler(request):

    ws = web.WebSocketResponse()
    await ws.prepare(request)

    async for msg in ws:
        if msg.type == aiohttp.WSMsgType.TEXT:
                ws.send_str('/answer')
    return ws
````
In our case:
```python
async def sse_handler(request):

    sse = EventSourceResponse()
    await sse.prepare(request)

    async with sse:
          sse.send('/answer')
    return sse
````